tensorflow/python/keras/engine/training.py

```diff
@@ -722,6 +722,8 @@ def fit(self,
               - tuple `(x_val, y_val)` of Numpy arrays or tensors
               - tuple `(x_val, y_val, val_sample_weights)` of Numpy arrays
               - dataset
+              - A generator or `keras.utils.Sequence` returning `(inputs, targets)`
+            or `(inputs, targets, sample_weights)`.
             For the first two cases, `batch_size` must be provided.
             For the last case, `validation_steps` could be provided.
             Note that `validation_data` does not support all the data types that
```

Comment:
Please remove this statement which is no longer accurate.

Reply:
which statement @fchollet ?
And the whole pull request is a single line, if  that line is now inadequate close the pull request?\

(plus, this simple PR was made 6 months ago, and was failed to be merged, that suggests a necessary review of your teams process @gbaned )